test/utils/utils.go

```diff
@@ -247,3 +256,51 @@ func UncommentCode(filename, target, prefix string) error {
 	// nolint:gosec
 	return os.WriteFile(filename, out.Bytes(), 0644)
 }
+
+// LoadImage does not work when using podman, so LoadImageArchive is used instead
+// refer to https://github.com/kubernetes-sigs/kind/issues/2038
+func LoadContainerImageToKindCluster(ctx context.Context,
+	kindCluster *kind.Cluster,
+	dockerImage string,
```

Comment:
```suggestion
	imageName string,
```

Reply:
Also please remove the existing LoadImageToKindClusterWithName if it isn't used  
https://github.com/etcd-io/etcd-operator/blob/16920cb54a6ecf4d4a429b833d97f23765c5f24d/test/utils/utils.go#L176